.github/workflows/main.yaml

```
@@ -38,13 +27,24 @@ jobs:
         run: echo ${{ secrets.GITLAB_ACCESS_TOKEN }} | docker login registry.git.equinor.com/sentry/decimate -u ${{ secrets.GITLAB_USERNAME }} --password-stdin
       - name: Build
         run: docker build . --tag ghcr.io/equinor/ccs-passive --cache-from ghcr.io/equinor/ccs-passive
-      - name: Push
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.x'
+      - name: "pip install pre-commit and poetry"
+        run: pip install pre-commit poetry
+      - name: build decimator
+        run: docker-compose -f docker-compose.ci.yaml build decimator
+      - name: Pre-Commit
+        run: pre-commit run --all-files --verbose -c .pre-commit-config.yaml
```

```suggestion
        run: pre-commit run --all-files --verbose
```
Not needed when using "default" pre-commit file